conference.js

```diff
@@ -316,6 +316,9 @@ export default {
         this.roomName = options.roomName;
         JitsiMeetJS.setLogLevel(JitsiMeetJS.logLevels.TRACE);
 
+        // attaches global error handler
+        window.onerror = JitsiMeetJS.getGlobalOnErrorHandler;
```

Comment:
Maybe it's good to check here if "JitsiMeetJS.getGlobalOnErrorHandler" exists.

Also I think it will be good to save the current window.onerror function and call it.